src/main/io/gps.c

```diff
@@ -683,7 +685,17 @@ void gpsInitUblox(void)
                         }
                         break;
                     case 12:
-                        ubloxSetNavRate(0x64, 1, 1); // set rate to 10Hz (measurement period: 100ms, navigation rate: 1 cycle) (for 5Hz use 0xC8)
+                        switch (gpsConfig()->gps_update_rate) {
```

Comment:
I think I want to just use an int to store the update rate.
The easiest way to do that would be to store it as a _period_ in milliseconds, because that can be passed straight to the existing `ubloxSetNavRate` function.
The other way, which would be easier for the user, would be to store it as a rate in Hz and have `ubloxSetNavRate` convert the Hz to a period. The problem here is that the rates may not be integers. The max rate I found to work on my units was 53ms, or 18.86Hz. So then we would set the max allowed value to 19Hz, but clamp the calculated period to 53ms.

Reply:
IMO using enum is a bit better here. It will prevent users using values that don't work. Without error message, this can be support nightmare.